node/src/node_configurator/node_configurator_standard.rs

```diff
@@ -737,6 +736,295 @@ mod tests {
         assert_eq!(config.crash_point, CrashPoint::Panic);
     }
 
+    fn fill_up_config_file(mut config_file: File) {
+        {
+            config_file
+                .write_all(b"real-user = \"1002:1002:/home/wooga\"\n")
+                .unwrap();
+            config_file
+                .write_all(b"blockchain-service-url = \"https://www.mainnet2.com\"\n")
+                .unwrap();
+            config_file
+                .write_all(b"clandestine-port = \"7788\"\n")
+                .unwrap();
+            config_file.write_all(b"consuming-private-key = \"00112233445566778899AABBCCDDEEFF00112233445566778899AABBCCDDEEFF\"\n").unwrap();
+            config_file.write_all(b"crash-point = \"None\"\n").unwrap();
+            config_file
+                .write_all(b"dns-servers = \"5.6.7.8\"\n")
+                .unwrap();
+            config_file
+                .write_all(b"earning-wallet = \"0xaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa\"\n")
+                .unwrap();
+            config_file.write_all(b"gas-price = \"77\"\n").unwrap();
+            config_file.write_all(b"log-level = \"trace\"\n").unwrap();
+            config_file
+                .write_all(b"mapping-protocol = \"pcp\"\n")
+                .unwrap();
+            config_file.write_all(b"min-hops = \"6\"\n").unwrap();
+            config_file
+                .write_all(b"neighborhood-mode = \"zero-hop\"\n")
+                .unwrap();
+            config_file.write_all(b"scans = \"off\"\n").unwrap();
+            config_file.write_all(b"rate-pack = \"2|2|2|2\"\n").unwrap();
+            config_file
+                .write_all(b"payment-thresholds = \"3333|55|33|646|999|999\"\n")
+                .unwrap();
+            config_file
+                .write_all(b"scan-intervals = \"111|100|99\"\n")
+                .unwrap()
+        }
+    }
+
+    #[test]
+    fn server_initializer_collected_params_handle_config_file_from_environment_and_real_user_from_config_file_with_path_started_by_dot() {
+        running_test();
+        let home_dir = ensure_node_home_directory_exists( "node_configurator_standard","server_initializer_collected_params_handle_config_file_from_environment_and_real_user_from_config_file_with_path_started_by_dot");
+        let data_dir = &home_dir.join("data_dir");
+        let config_file_relative = File::create(PathBuf::from("./generated").join("config.toml")).unwrap();
+        fill_up_config_file(config_file_relative);
+        let env_vec_array = vec![
+            ("MASQ_CONFIG_FILE", "./generated/config.toml"),
+            #[cfg(not(target_os = "windows"))]
+            ("MASQ_REAL_USER", "9999:9999:booga"),
```

Comment:
What role does this parameter plays here? If none big one, you should remove it. If some real than think about what will happen for this test when running on Windows... It will be missing.

Bu the way, Windows doesn't need to deal with this business around the real-user, never. Maybe your tests, and now I really talk in a plural, should not run on Windows completely, or some of these??? Do think about that please.